Add projects checklist (#313)
open-energy-transition/handbook: docs/Engineering/Projects.mdx

import MaintainerTag from "@site/src/components/MaintainerTag";
import ControlledDocBanner from "@site/src/components/ControlledDocBanner";

<MaintainerTag maintainerEmails={["hazem.abdelkhalek@openenergytransition.org"]} />

# Project Management Checklist

<ControlledDocBanner />

## Project secured, what now?

As a project director, you must:

### Project management:
- Create a discord thread numbered correctly.
- Create a google drive folder numbered correctly.
- Add proposal, contract.. similar document in the folder.
- Update the project status on hubspot. (If no access, reach out to Head of Market Dev.)

> - Create a project on OpenProject (soon)

Reviewer: Can we add links here

### Staffing & timetracking:

> - Create a tab in the project planning sheet.

Reviewer: Link

- Initiate staffing process through the relevant HoD. (if not done already).
- Contact Finance dept. to confirm the feasibility of the staffing plan.

> - Add project to dropdown in the timetracking sheet.

Reviewer: Link

### Communication
- Project director must be added to the project directors google group.

> - Make sure you're invited to the project sync meeting.

Reviewer 2: ```suggestion
- Make sure you're invited to the monthly project sync meeting.
```

- Announce the project start in the following formal all hands meeting.

### Security 

> - Conduct the risk assesemnet and fill in the template document.

Reviewer 2: ```suggestion
- Conduct the risk assessment and fill in the template document.
```

## Project finished, what now?

As a project director, you must:

### Project Management.

> - All reports, presentations, meeting minutes..etc correctly placed in the google drive.

Reviewer 2: ```suggestion
- All reports, presentations, meeting minutes, etc correctly placed in the google drive.
```

- Repository clearly documented and tested for reproducibility (if applicable)
--> general rule: document everything to the point that make you able pick up where you left off even after a year.
- Hide project tab in the project planning sheet.

### Communication
- Make sure to discuss follow up ideas in the last client meeting.

> - Hold a project debrief session according to the template.

Reviewer 2: ```suggestion
- Hold a project [retrospective meeting](RetrospectiveMeeting) according to the template.
```
